app/forms/asylum_support_form.rb

```diff
@@ -0,0 +1,10 @@
+class AsylumSupportForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include SessionPersistable
+
+  ATTRIBUTES = %i[asylum_support].freeze
+
+  attribute :asylum_support, :boolean
```

Comment:
We also need to send this boolean to CFE as an applicant attribute (although CFE will only know what to do with it when https://github.com/ministryofjustice/check-financial-eligibility-partner/pull/89 is merged)